motor_controller:
  ros__parameters:
    serial_address: '/dev/ttyACM1'
    pwm_pins: [3, 5, 9, 11]
    dir_pins: [4, 6, 10, 12]
    max_speed: 1.0

drive_control:
  ros__parameters:
    default_linear_speed: 0.5
    default_angular_speed: 0.3

motor_health_monitor:
  ros__parameters:
    check_interval: 5
    current_threshold: 10.0
    temperature_threshold: 80.0
